src/graphics/program-lib/chunks/lit/frag/fresnelSchlick.js

```diff
@@ -1,14 +1,14 @@
 export default /* glsl */`
 // Schlick's approximation
 vec3 getFresnel(float cosTheta, vec3 f0) {
-    float fresnel = pow(1.0 - abs(cosTheta), 4.0);
-    float glossSq = dGlossiness * dGlossiness;
-    return f0 + (1.0 - f0) * fresnel * glossSq;
+    float fresnel = pow(1.0 - max(cosTheta, 0.0f), 5.0);
+    float glossSq = (dGlossiness * dGlossiness);
+    return f0 + (max(vec3(glossSq), f0) - f0) * fresnel;
```

Comment:
I wonder if some comments here would be useful to understand what is happening here.